xml_inscriptions/transcribed/DC.DO.GL.35.4.xml

```diff
@@ -110,7 +110,7 @@
    
    <text>
       <body>
-       <div type="edition>
+      
```

Comment:
Why is this being removed? I think the broader issue is that a version of this file is still housed in the metadata folder, which is why this isn't displaying correctly.